scripts/scripts.js

```diff
@@ -145,6 +145,26 @@ const CONFIG = {
   atvCaptionsKey: 'bacom',
 };
 
+export const EVENT_LIBS = (() => {
```

Comment:
Can we put this lower down in the file to prioritize lcp, etc? Maybe right above `loadPage`